tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -1091,7 +1094,10 @@ def test_task_one_granule_4_files_success(
         }
 
         mock_s3_cli = mock_boto3_client("s3")
-        mock_s3_cli.restore_object.side_effect = [None, None, None, None]
+        mock_s3_cli.restore_object.side_effect = [{"ResponseMetadata": {"HTTPStatusCode": 200}},
```

Comment:
The status value should be 202 on an initial restore. 200 occurs if the granule has already been restored.

Reply:
Fixed